src/NzbDrone.Core.Test/MediaFiles/EpisodeImport/ImportDecisionMakerFixture.cs

```diff
@@ -226,10 +233,100 @@ public void should_not_throw_if_episodes_are_not_found()
                 .Setup(c => c.FilterExistingFiles(_videoFiles, It.IsAny<Series>()))
                 .Returns(_videoFiles);
 
-            Subject.GetImportDecisions(_videoFiles, _series, false);
+            Subject.GetImportDecisions(_videoFiles, _series);
+
+            Mocker.GetMock<IParsingService>()
+                  .Verify(c => c.GetLocalEpisode(It.IsAny<String>(), It.IsAny<Series>(), It.IsAny<Boolean>(), It.IsAny<ParsedEpisodeInfo>()), Times.Exactly(_videoFiles.Count));
+        }
+
+        [Test]
+        public void should_not_use_folder_for_full_season()
+        {
+            var videoFiles = new[]
+                             {
+                                 @"C:\Test\Unsorted\Series.Title.S01\S01E01.mkv".AsOsAgnostic(),
+                                 @"C:\Test\Unsorted\Series.Title.S01\S01E02.mkv".AsOsAgnostic(),
+                                 @"C:\Test\Unsorted\Series.Title.S01\S01E03.mkv".AsOsAgnostic()
+                             };
+
+            GivenSpecifications(_pass1);
+            GivenVideoFiles(videoFiles.ToList());
```

Comment:
make this method take `IEnumarble` and call `.ToList()` in it. (would look nicer, less needy)